src/test/java/org/prebid/server/ApplicationTest.java

```diff
@@ -111,73 +115,209 @@ public static void setUp() throws IOException {
-
+    public void openrtb2AuctionShouldRespondWithBidsFromConversant() throws IOException, JSONException {
+        //given
         // conversant bid response for imp 4
         wireMockRule.stubFor(post(urlPathEqualTo("/conversant-exchange"))
-                .withRequestBody(equalToJson(jsonFrom("openrtb2/test-conversant-bid-request-1.json")))
-                .willReturn(aResponse().withBody(jsonFrom("openrtb2/test-conversant-bid-response-1.json"))));
+                .withRequestBody(equalToJson(jsonFrom("openrtb2/conversant/test-conversant-bid-request-1.json")))
+                .willReturn(aResponse().withBody(jsonFrom("openrtb2/conversant/test-conversant-bid-response-1.json"))));
 
         // conversant bid response for imp 4 with alias parameters
         wireMockRule.stubFor(post(urlPathEqualTo("/conversant-exchange"))
-                .withRequestBody(equalToJson(jsonFrom("openrtb2/test-conversant-bid-request-2.json")))
-                .willReturn(aResponse().withBody(jsonFrom("openrtb2/test-conversant-bid-response-2.json"))));
+                .withRequestBody(equalToJson(jsonFrom("openrtb2/conversant/test-conversant-bid-request-2.json")))
+                .willReturn(aResponse().withBody(jsonFrom("openrtb2/conversant/test-conversant-bid-response-2.json"))));
 
-        // facebook bid response for imp 5
-        wireMockRule.stubFor(post(urlPathEqualTo("/facebook-exchange"))
-                .withRequestBody(equalToJson(jsonFrom("openrtb2/test-facebook-bid-request-1.json")))
-                .willReturn(aResponse().withBody(jsonFrom("openrtb2/test-facebook-bid-response-1.json"))));
+        // pre-bid cache
+        wireMockRule.stubFor(post(urlPathEqualTo("/cache"))
+                .withRequestBody(equalToJson(jsonFrom("openrtb2/conversant/test-cache-conversant-request.json")))
+                .willReturn(aResponse().withBody(jsonFrom("openrtb2/conversant/test-cache-conversant-response.json"))));
+
+        //when
+        final Response response = given(spec)
+                .header("Referer", "http://www.example.com")
+                .header("X-Forwarded-For", "192.168.244.1")
+                .header("User-Agent", "userAgent")
+                .header("Origin", "http://www.example.com")
+// 		           this uids cookie value stands for
+//                {"uids":{"conversant":"CV-UID"}}
+                .cookie("uids", "eyJ1aWRzIjp7ImNvbnZlcnNhbnQiOiJDVi1VSUQifX0=")
+                .body(jsonFrom("openrtb2/conversant/test-auction-conversant-request.json"))
+                .post("/openrtb2/auction");
 
+        //then
+        final Map<String, String> exchanges = new HashMap<>();
+        exchanges.put(CONVERSANT, "http://localhost:" + WIREMOCK_PORT + "/conversant-exchange");
+        exchanges.put(CONVERSANT_ALIAS, null);
+
+        final String expectedAuctionResponse = auctionResponseFrom(jsonFrom("openrtb2/conversant/test-auction-conversant-response.json"),
```

Comment:
I think it would be better to create two specializations of this method for different types of responses - legacy /auction and /openrtb2/auction along the lines of:
```
legacyAuctionResponseFrom(String template, Response response, Map<String, String> exchanges) {
    return auctionResponseFrom(template, response, "ext.responsetimemillis.%s", )
}
```
and similar for openrtb auction response and use them in different test methods depending on endpoint type their are testing against.